### PR TITLE
Created Endpoints for CRUD

### PR DESCRIPTION
**Pull request by tak074**

### CODE REVIEW

**Comment by tak074** [comment]
@YouFaraj  pull request please!

File: Server/server.js [hunk 1]
```diff
@@ -1,44 +1,96 @@
...
+
+app.put('/products/:product/', function (req, res) {
+  var productParam = req.params.product;
+  var product = req.body.product;
+  var imageName = req.body.imageName;
+  var color = req.body.color;
+  var url = req.body.url;
+  var alt = req.body.alt;
+
+  var updates = {};
```
Comment (YouFaraj) [inline Server/server.js]: Pretty neat!

File: Server/server.js [hunk 2]
```diff
@@ -64,16 +64,23 @@ app.put('/products/:product/', function (req, res) {
   var url = req.body.url;
   var alt = req.body.alt;
 
-  var updates = {};
-  for (var key in req.body) {
-    updates.key = req.body.key;
-  }
-
-  Image.updateOne({product: productParam}, updates, function(err, result) {
```
Comment (YouFaraj) [inline Server/server.js]: Is this supposed to update existing data or create a new one if it doesn't?

**Review by YouFaraj — APPROVED** [review]
Looks good so far, I'd suggest adding brief comments above the routes to describe their desired outcome. Also left a comment for the app.put but it's not an error just wondering. Great work though!

**Pull request merged**